### ISSUE TITLE
Continue reading entries broken

### ISSUE THREAD

**Issue by Borewit** [feross/drag-drop#52]
It looks like the recursion, to read more _entries_ is called with missing argument reader.

I am getting the following error:
```
Uncaught TypeError: Cannot read property 'readEntries' of undefined
    at readEntries (C:\Users\Borewit\code\github\drag-drop\index.js:176)
    at C:\Users\Borewit\code\github\drag-drop\index.js:179
```
original issue: [webtorrent-desktop#1771](https://github.com/webtorrent/webtorrent-desktop/issues/1771)

https://github.com/feross/drag-drop/blob/89fd6bdeb63ba19493e658b27058dca127953ef4/index.js#L175-L184

I think like 179 should be:
```js
readEntries(reader)
```

PR: ~~#53~~ #51

**Comment by nstephenh** [comment]
I would like to use drag-drop for dropping in a directory into the browser, is there any update on this?

**Comment by Borewit** [comment]
Should be fixed by merging PR #51 @feross.
